src/DotNetWorker.Grpc/GrpcWorker.cs

```diff
@@ -35,7 +35,8 @@ internal class GrpcWorker : IWorker
         private readonly IOutputBindingsInfoProvider _outputBindingsInfoProvider;
         private readonly IInputConversionFeatureProvider _inputConversionFeatureProvider;
         private readonly IMethodInfoLocator _methodInfoLocator;
-        private readonly IOptions<GrpcWorkerStartupOptions> _startupOptions;
```

Comment:
I don't know that we had to change this item IOptions<GrpcWorkerStartupOptions> but I'm not the best person to ask xD

cc: @kshyju @jviau

Reply:
Discussed with Jacob offline - didn't need to but changing to create consistency with WorkerOptions